pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Problem
+- *Description of why you made this PR*
+
+## Solution
+- *And how do you fix that problem*
+
+## PR Status
+- [ ] Code finished.
+- [ ] Tested with Package_check.
+- [ ] Fix or enhancement tested.
+- [ ] Upgrade from last version tested.
+- [ ] Can be reviewed and tested.
+
+## Validation
+---
+- [ ] **Code review**
+- [ ] **Approval (LGTM)**  
+*Code review and approval have to be from a member of @YunoHost/apps group*
+- **CI succeeded** : 
+[![Build Status](https://ci-apps-dev.yunohost.org/jenkins/job/phpldapadmin_ynh%20-BRANCH-/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/phpldapadmin_ynh%20-BRANCH-/)  
```

Comment:
As well, you can't expect any automatic tests for this app.

Reply:
I agree but ci-apps-dev is not automatic, having the link already setuped allows to easily have the results of a test launched manually on ci-apps-dev. it's a bad idea ?

Reply:
This link won't work as it is. So for me it's not really useful.
The link will without the branch, but will the name of the person who add the job.
Also, unless you're the maintainer, I doubt anyone would use it.

Reply:
ok then, as I don't plan to be the maintainer, I removed it ^^